### ISSUE TITLE
set_missing_count treat USUBJID as a factor

### ISSUE THREAD

**Issue by om9391** [atorus-research/Tplyr#83]
### Description

set_missing_count() currently counts the NA's actually present in the data.
I'd like to display the missing count as the number of subjects who don't have an assessment at the current visit. 
This could be mocked up in the input data by joining a mock dataset of subjects, PARAMs, and AVISIT values so that the NA's insert themselves, but can Tplyr do this automatically?

### Example of Table (Feature Request Only)

For example below, that would be Missing = 1 for TRT B for each PARAM and AVISIT.

``` r
library(Tplyr)
library(tibble)
library(dplyr)

adsl <-
  tribble(~ USUBJID, ~ SAFFL, ~TRT01AN, ~TRT01A,
          "101-001", "Y",     1,        "TRT A",
          "101-002", "Y",     2,        "TRT B",
          "101-003", "Y",     1,        "TRT A",
          "101-004", "Y",     2,        "TRT B")

adpe <- tribble(~USUBJID,  ~AVISITN, ~AVISIT,     ~PARAMN, ~PARAM,   ~TRT01AN, ~TRT01A,    ~AVALC,   ~SAFFL,
                "101-001",  -10,      "Screening",  1,      "Head",    1,     "TRT A",  "Normal", "Y",
                "101-001",  -10,      "Screening",  2,      "Lungs",   1,     "TRT A",  "Normal", "Y",
                "101-001",  -1,       "Day -1",     2,      "Lungs",   1,     "TRT A",  "Normal", "Y",
                "101-001",   5,       "Day 5",      2,      "Lungs",   1,     "TRT A",  "Normal", "Y",
                
                "101-002",  -10,      "Screening",  1,      "Head",    2,     "TRT B",  "Normal", "Y",
                "101-002",  -10,      "Screening",  2,      "Lungs",   2,     "TRT B",  "Normal", "Y",
                "101-002",  -1,       "Day -1",     2,      "Lungs",   2,     "TRT B",  "Normal", "Y",
                "101-002",   5,       "Day 5",      2,      "Lungs",   2,     "TRT B",  "Normal", "Y",
                
                "101-003",  -10,      "Screening",  1,      "Head",    1,     "TRT A",  "Normal", "Y",
                "101-003",  -10,      "Screening",  2,      "Lungs",   1,     "TRT A",  "Abnormal", "Y",
                "101-003",  -1,       "Day -1",     2,      "Lungs",   1,     "TRT A",  "Normal", "Y",
                "101-003",   5,       "Day 5",      2,      "Lungs",   1,     "TRT A",  "Abnormal", "Y",
)

adpe$AVALC <- factor(adpe$AVALC, levels = c("Normal", "Abnormal"))

t <- tplyr_table(adpe, TRT01A, where = SAFFL == 'Y') %>%
  set_pop_data(adsl) %>%
  set_pop_treat_var(TRT01A) %>%
  set_pop_where(SAFFL =="Y") %>%
  add_layer(
    group_count(AVALC, by = vars(PARAM, AVISIT)) %>%
     set_distinct_by(USUBJID) %>%
     set_denoms_by(TRT01A) %>%
     set_missing_count(f_str("xxx", n), denom_ignore=FALSE, Missing = NA)
  ) 


dat<- build(t) %>%
  arrange(ord_layer_1, ord_layer_2, ord_layer_3)

```

**Comment by mstackhouse** [comment]
I think the most straightforward way to do this is to mirror the implementation of `add_total_row()`. So the function could be `add_missing_subjects_row()`. The logic of the implementation should be vaguely similar:

- Specify to create a row and provide format
- Process the data and do an extra specific action
- Format it according to user supplied format

**Comment by mstackhouse** [comment]
Closed via #174
